lnwallet/btcwallet/btcwallet.go

```diff
@@ -1023,16 +1023,26 @@ func minedTransactionsToDetails(
 			destAddresses = append(destAddresses, outAddresses...)
 		}
 
+		var previousOutPoints []wire.OutPoint
+		for _, txIn := range wireTx.TxIn {
+			previousOutPoint := wire.OutPoint{
+				Hash:  txIn.PreviousOutPoint.Hash,
```

Comment:
`txIn.PreviousOutPoint` already is of type `wire.OutPoint`, so no need to create a new struct. This can just be `previousOutPoints[idx] = txIn.PreviousOutPoint` (if you also apply the same comment about pre-allocation and index-based access here).